test/testcafe/spec/StateTokenRefreshUseCases_spec.js

```diff
@@ -0,0 +1,75 @@
+import { RequestMock, RequestLogger } from 'testcafe';
+import SelectAuthenticatorPageObject from '../framework/page-objects/SelectAuthenticatorPageObject';
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import xhrIdentify from '../../../playground/mocks/data/idp/idx/identify';
+import xhrErrorIdentify from '../../../playground/mocks/data/idp/idx/error-identify-access-denied';
+import xhrAuthenticatorVerifySelect from '../../../playground/mocks/data/idp/idx/authenticator-verification-select-authenticator';
+
+const identifyMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentify)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
+  .respond(xhrErrorIdentify, 403);
+
+const identifyThenSelectAuthenticatorMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentify)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
+  .respond(xhrAuthenticatorVerifySelect);
+
+const identifyRequestLogger = RequestLogger(
+  /idx\/identify/,
+  {
+    logRequestBody: true,
+    stringifyRequestBody: true,
+  }
+);
+
+fixture('Statetoken Page Refresh');
+
+async function setup(t) {
+  const identityPage = new IdentityPageObject(t);
+  await identityPage.navigateToPage();
+  return identityPage;
+}
+
+async function setupA(t) {
+  const identityPage = new SelectAuthenticatorPageObject(t);
+  await identityPage.navigateToPage();
+  return identityPage;
+}
+
+test.requestHooks(identifyRequestLogger, identifyMock)('should set and remove statetoken in sessionStorage', async t => {
```

Comment:
you don't need `identifyRequestLogger` if you didn't utilize logger result in the test.